src/sentry/static/sentry/app/views/organizationEventsV2/data.jsx

```diff
@@ -64,6 +64,28 @@ export const ALL_VIEWS = deepFreeze([
     ],
     columnWidths: ['3fr', '70px', '70px', '1fr', '1.5fr'],
   },
+  {
+    id: 'transactions',
+    name: 'Transactions',
+    data: {
+      fields: ['event', 'type', 'project', 'user', 'time'],
+      orderby: ['-timestamp', '-id'],
+      query: 'event.type:transaction',
+    },
+    tags: [
+      'event.type',
```

Comment:
Won't this always be 100% because of the conditions?

Reply:
Yea I'm going to adjust this. I just copy/pasted the default view.

Reply:
This will be addressed in https://github.com/getsentry/sentry/pull/14103